src/octopeer-github/main/ElementEventBinding/ClickElementEventBinding.ts

```diff
@@ -1,13 +1,11 @@
+/// <reference path="../ElementSelectionBehaviour/ElementSelectionBehaviour.ts"/>
 /**
  * Created by Youri on 04/05/2016.
- */
-
-/**
  * Class for adding click events.
  */
 class ClickElementEventBinding implements ElementEventBinding {
-    private eventType: string = "click";
-    private eventID: EventID  = new EventID(1);
+    public eventType: string = "click";
+    public eventID: EventID  = new EventID(101);
```

Comment:
Why are all `EventID`s and `ElementID`s in the range of 100? I'm assuming they will be in the database with IDs starting from 1. But in any case, the database isn't finished yet, so then these can serve as placeholders